test/lib/print-test.js

```diff
@@ -90,11 +90,9 @@ test('print: prints verbs in cyan', t => {
 })
 
 test('print: prints texts in cyan', t => {
```

Comment:
The `Text` objects were bold-cyan, but they are now `yellow` according to `print.js`.  Is that a mistake?  Do you want to switch them to yellow or keep them as bold cyan?